scimath/units/tests/sample_units.py

```diff
@@ -0,0 +1,9 @@
+
+from scimath.units.length import mm
+from copy import copy
+
+
+custom_unit = copy(mm)
+
+custom_unit_with_label = copy(mm)
+custom_unit_with_label.label = "cuwl"
```

Comment:
nit: why do we need two units for the test below ? just `custom_unit.label = 'cuwl'` should be sufficient..